src/vellum/workflows/outputs/base.py

```diff
@@ -32,7 +32,7 @@ def __init__(
         if value is not undefined and delta is not undefined:
             raise ValueError("Cannot set both value and delta")
 
-        self._name = name
+        self._name = name.replace("-", "_")  # Convert hyphens to underscores for valid python variable names
```

Comment:
[oos] We likely need to reproduce the full sanitization method that we have in codegen here. Probably as its own util. 

[way oos] Once we move display context from vembda initiated -> workflow initiated, I think we should be able to use that mapping instead of reproducing the logic. This is a refactor that I'm tracking this sprint